### DISCUSSION TITLE
Getting the replicad-app-example to work in nextjs

### DISCUSSION THREAD

**Discussion by kriho** [sgenoud/replicad#140] [Q&A] [answered]
Hi everyone,
I am trying to set up an ambitious project with replicad and nextjs.
Since I have little control over how to load the wasm I wanted to ask if anyone has already seen or solved this issue.

I used a starter project for nextjs and added the relevant files from the replicad-app-example package.
I also enabled the "asyncWebAssembly" feature for webpack in the "next.config.js".

When I run `next build` I get the following error:
```
Failed to compile.

./node_modules/replicad-opencascadejs/src/replicad_single.wasm?url
Module not found: Can't resolve 'a'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./app/worker.js
./app/App.tsx
```

What am I missing? Thanks!

**Comment by sgenoud** [comment]
Sorry I have not worked with next at all, I will write here if I try it at some point!

**Reply by kriho** [reply]
Okay, thanks!

**Reply by kriho** [reply, marked as answer]
Solved here https://github.com/sgenoud/replicad/discussions/79#discussioncomment-5816019
